gengokit/httptransport/httptransport.go

```diff
@@ -95,6 +95,10 @@ func NewBinding(i int, meth *svcdef.ServiceMethod) *Binding {
 			newField.IsBaseType = true
 		}
 
+		if !newField.IsBaseType {
```

Comment:
Hmm, I think I prefer bringing this up to line 96 as an `else` block. 

It then convays the information. I am checking if this is not a struct type, if it is not a struct type then it is a base type, otherwise we add `pb.` to the beginning of `newField.GoType.`